### PR TITLE
📖 Fix dead link in AMP dev docs

### PR DESCRIPTION
**Pull request by rsimha**
Our dev docs point to the codelab at https://codelabs.developers.google.com/codelabs/creating-your-first-amp-component/index.html as a good starting point for writing a new AMP component.

image

The codelab has disappeared and the link 404s, so this PR replaces it with [this guide](https://github.com/ampproject/amphtml/blob/master/contributing/building-an-amp-extension.md) wherever applicable.

### CODE REVIEW

File: contributing/DEVELOPING.md [hunk 1]
```diff
@@ -35,7 +35,6 @@ Before you start writing code, consult these resources for guidance and guidelin
 - **CSS**: [Writing CSS For AMP Runtime](writing-css.md)
 - **Creating new components**:
   - [Instructions and Guidelines for building an AMP component](building-an-amp-extension.md)
-  - Learn to create your first component in this [codelab](https://codelabs.developers.google.com/codelabs/creating-your-first-amp-component/#0)
```
Comment (caroqliu) [inline contributing/DEVELOPING.md:38]: Why is this removed instead of updated as it is in `contributing-code.md`?

**Review by caroqliu — APPROVED** [review]

File: contributing/DEVELOPING.md (hunk 1 above)
Comment (rsimha) [reply contributing/DEVELOPING.md:38]: I was under the impression the preferred link was `building-an-amp-extension.md`, which is already being mentioned in the line above. Does that work?

Comment (caroqliu) [reply contributing/DEVELOPING.md:38]: Ah yes, thanks.

**Review by kristoferbaxter — APPROVED** [review]
